commit 8d5b0e6cf47dc5abf66cb6fb55df51958db97b3f
Author: Cameron Nemo <camerontnorman@gmail.com>
Date:   Fri Feb 8 13:38:23 2019 -0800

    apparmor: allow old config file location
    
    apparmor: install local modification file

diff --git Makefile.am Makefile.am
index 7e58f08..d9922da 100644
--- Makefile.am
+++ Makefile.am
@@ -42,6 +42,9 @@ APPDATA_IN_FILES = \
 APPARMOR_IN_FILES = \
 	data/apparmor/usr.bin.redshift.in
 
+APPARMOR_LOCAL_FILES = \
+	data/apparmor/local/usr.bin.redshift
+
 
 # Icons
 if ENABLE_GUI
@@ -109,7 +112,9 @@ endif
 # AppArmor profile
 if ENABLE_APPARMOR
 apparmordir = @sysconfdir@/apparmor.d
+apparmorlocaldir = $(apparmordir)/local
 apparmor_DATA = $(APPARMOR_IN_FILES:.in=)
+apparmorlocal_DATA =  $(APPARMOR_LOCAL_FILES)
 
 $(apparmor_DATA): $(APPARMOR_IN_FILES) Makefile
 	$(AM_V_GEN)$(MKDIR_P) $(@D) && \
diff --git data/apparmor/local/usr.bin.redshift data/apparmor/local/usr.bin.redshift
new file mode 100644
index 0000000..e735411
--- /dev/null
+++ data/apparmor/local/usr.bin.redshift
@@ -0,0 +1 @@
+# Site-specific additions and overrides for 'usr.bin.redshift'
diff --git data/apparmor/usr.bin.redshift.in data/apparmor/usr.bin.redshift.in
index dd9d9ae..644c64c 100644
--- data/apparmor/usr.bin.redshift.in
+++ data/apparmor/usr.bin.redshift.in
@@ -35,7 +35,8 @@
   # Allow but log any other dbus activity
   audit dbus bus=system,
 
-  owner @{HOME}/.config/redshift/redshift.conf r,
+  owner @{HOME}/.config/{,redshift/}redshift.conf r,
+  owner /{,var/}run/user/[0-9]*/redshift-shared-* rw,
 
   # Site-specific additions and overrides. See local/README for details.
   #include <local/usr.bin.redshift>
